js/google-shopping.js

```diff
@@ -1,3 +1,134 @@
 // Write your solutions below
 
+//PART 1
+var count = 0;
```

Comment:
i like how far ahead you are into the exercise but some comments to explain what you do would be nice